CMakeLists.txt

```diff
@@ -205,6 +207,98 @@ if (NN_ENABLE_NANOCAT)
     target_link_libraries (nanocat ${PROJECT_NAME})
 endif ()
 
+find_program (ASCIIDOCTOR_EXE asciidoctor)
+if (NOT ASCIIDOCTOR_EXE)
+    message (WARNING "asciidoctor not found, skipping docs")
+    set (NN_ENABLE_DOC OFF)
+else ()
+    if (NN_ENABLE_DOC)
+        message (STATUS "Using asciidoctor at ${ASCIIDOCTOR_EXE}")
```

Comment:
It was very helpful .. I can remove it now if you like.  (Problem was that I couldn't tell whether this was working or not on the VS builds because VS output sucks.)